packages/decentraland-ecs/src/setupProxy.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path')
+const fs = require('fs')
+
+module.exports = function (dcl, app, express) {
+    const dclKernelPath = path.resolve(dcl.getWorkingDir(), 'node_modules', 'decentraland-kernel');
+    const dclKernelDefaultProfilePath = path.resolve(dclKernelPath, 'default-profile');
+    const dclUnityRenderer = path.resolve(dclKernelPath, 'unity-renderer');
```

Comment:
Same as `decentraland-kernel`

Reply:
I only changed the dclKernelPath because the others are relative to it. If decentraland-kernel is resolved, can we assume the others exist?

Reply:
I think this one should be assumed to exist also, resolved the same way for consistency
```suggestion
    const dclUnityRenderer = path.dirname(require.resolve('@dcl/unity-renderer/package.json', {paths: [dcl.getWorkingDir()]}));
```